.github/workflows/deploy.yml

```diff
@@ -13,6 +13,13 @@ jobs:
     name: Deploy docs to GitHub Pages
     runs-on: ubuntu-latest
     steps:
+      - name: Generate token
+        id: generate_token
+        uses: tibdex/github-app-token@b62528385c34dbc9f38e5f4225ac829252d1ea92
```

Comment:
Why do we fix the action to this exact version? Does it have any security implications to use a 3rd party action for working with the app-id and private-keys?

Reply:
Yep, these [github docs examples](https://docs.github.com/en/issues/planning-and-tracking-with-projects/automating-your-project/automating-projects-using-actions) suggest using actions pinned to specific versions.